packages/net/http/_client_connection.pony

```diff
@@ -189,9 +192,11 @@ actor _ClientConnection is HTTPSession
     """
     Close the connection from the client end.
     """
+    _cancel_all()
```

Comment:
The behaviour of cancelling all pending requests should be documented in the docstring.

Reply:
Will add it.

Reply:
Hi Matthias, I processed your remarks and pushed a new commit.